scripts/monitor/rano_monitor/widgets/subject_details.py

```diff
@@ -130,20 +138,23 @@ def update_subject(self):
 
     def __update_buttons(self):
         review_msg = self.query_one("#review-msg", Static)
+        annotations_disabled_msg = self.query_one("#annotations-disabled-msg", Static)
         review_button = self.query_one("#review-button", Button)
         reviewed_button = self.query_one("#reviewed-button", Button)
         brainmask_button = self.query_one("#brainmask-review-button", Button)
         valid_btn = self.query_one("#valid-btn", Button)
 
-        if is_editor_installed():
-            review_msg.display = "none"
-            review_button.disabled = False
-        if self.__can_finalize():
-            reviewed_button.label = "Mark as finalized"
-            reviewed_button.disabled = False
-        if self.__can_review_brain():
-            brainmask_button.label = "Review brain mask"
-            brainmask_button.disabled = False
+        if ANNOTATIONS_ENABLED:
+            annotations_disabled_msg.display = "none"
+            if is_editor_installed():
+                review_msg.display = "none"
+                review_button.disabled = False
```

Comment:
So you don't make this check if annotations are disabled, right? thus showing `review_msg`?
And also it looks like all the buttons cannot be disabled if not ANNOTATIONS_ENABLED?

Reply:
*fixing myself. Buttons ARE disabled by default and we cannot enable them back. So everything is OK

Reply:
*fix to fix:) everything is ok with button disabling, but the question about `review_msg`  still stands